packages/amplify-graphql-model-transformer/API.md

```diff
@@ -5,7 +5,10 @@
 ```ts
 
 import { AppSyncDataSourceType } from '@aws-amplify/graphql-transformer-interfaces';
+import { AwsCustomResource } from 'aws-cdk-lib/custom-resources';
```

Comment:
I'm surprised at the number of new exports from this package -- are these all only to enable unit testing of new helpers? If so, we should adjust our package-level `index.ts` to ensure that we only vend truly public exports from the package. If not, what other packages are using the new exports? (I'm assuming these aren't intended for customer consumption?)